prac_05/word_occurrences.py

```diff
@@ -0,0 +1,13 @@
+def main():
```

Comment:
I cannot see any problems big problems with the code - except there could be more comments and perhaps a doc string at the start !